MatrixKit/Views/RoomList/MXKRecentTableViewCell.m

```diff
@@ -44,7 +44,8 @@ - (void)render:(MXKCellData *)cellData
         }
         
         // Set in bold public room name
-        if (roomCellData.roomDataSource.room.state.isJoinRulePublic)
+        // @TODO
```

Comment:
Move isJoinRulePublic to MXRoomSummary to avoid to be dependent on room state.
This will be require when room state access will be async.